.packit.yml

```diff
@@ -9,3 +9,9 @@ jobs:
     trigger: pull_request
     targets:
       - fedora-all
+
+  - job: tests
+    trigger: pull_request
+    targets:
+      - fedora-all
+    skip_build: true
```

Comment:
This line is not necessary.  The test will automatically inherit the builds done in by the `copr_build` jobs.  The package will not be built twice.
```suggestion
```

Reply:
At also means, that the current TF jobs use the latest package from Fedora and not the build from Packit.

Reply:
Good catch, thanks